packages/next/src/server/lib/patch-fetch.ts

```diff
@@ -211,10 +211,12 @@ export function patchFetch({
     // Do create a new span trace for internal fetches in the
     // non-verbose mode.
     const isInternal = (init?.next as any)?.internal === true
+    const hideSpan = process.env.NEXT_OTEL_FETCH_DISABLED === '1'
```

Comment:
Let's make it a next config option?

Reply:
reasoning we did env instead was so that it could be configured by upstream tooling without requiring the user to modify their `next.config.js` manually

Reply:
can we document then?

Reply:
but lgtm otherwise